sdk/schemaregistry/azure-schemaregistry-client/src/main/java/com/azure/schemaregistry/client/rest/AzureSchemaRegistryRestServiceClientBuilder.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.schemaregistry.client.rest;
```

Comment:
This should be in `implementation` package as it's the convention we follow to denote packages that users shouldn't use.
https://azure.github.io/azure-sdk/java_design.html#example-namespaces

Reply:
Moved to implementation